pkg/kf/apps/kfapp.go

```diff
@@ -152,6 +152,21 @@ func (k *KfApp) DeleteEnvVars(names []string) {
 	k.SetEnvVars(envutil.RemoveEnvVars(names, k.GetEnvVars()))
 }
 
+// GetHealthCheck gets the readiness probe or nil if one doesn't exist.
+func (k *KfApp) GetHealthCheck() *corev1.Probe {
```

Comment:
Should we add some example tests for these in kfapp_test.go?

Reply:
done!